recipe/data.go

```diff
@@ -1713,19 +1713,19 @@ var AgeingPopProj = Response{
 	},
 }
 
-// Local authority ageing statistics - % of single households headed by someone over 65 and over 85
+// Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over
 var AgeingSingleHouseholds = Response{
 	ID:     "226d6940-ed0c-4556-91f1-9ca61d56c8a0",
-	Alias:  "Local authority ageing statistics - % of single households headed by someone over 65 and over 85",
+	Alias:  "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over",
 	Format: "v4",
 	InputFiles: []file{
 		{"AgeingSingleHouseholds"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ageing-single-households",
+			DatasetID: "older-people-in-single-households",
 			Editions:  []string{"time-series"},
-			Title:     "Local authority ageing statistics, percentage of single households headed by someone over 65 and over 85",
+			Title:     "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over",
```

Comment:
Should we use a comma here:
```suggestion
			Title:     "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over, and 85 and over",
```

Reply:
same as earlier